README.md

```diff
@@ -99,17 +99,18 @@ Using the iterators, note that we can get each cell population by looping throug
 2 levels below the top, so
 
 ```julia
-for cell in level_iter(embryo,2)
+for cell in level_iter(embryo,3)
   # Do something with the cells!
 end
 ```
 
 To apply a function cell-by-cell, you can write a dispatch `f` on the type for the
 level. Using `level_iter_idx`, we can have its changes update some other head node
-`d_embryo` via:
+`d_embryo` via: (((here, I think `d_embryo` is not defined, reader does not know what it is
```

Comment:
could you add a quick "this is what the function looks like" kind of thing that introduces those variables?